### DISCUSSION TITLE
KMS-v4.0.0

### DISCUSSION THREAD

**Discussion by KENNYSOFT** [KENNYSOFT/WzComparerR2#82] [General]
### UI
* '패치 도구', '그림', '시각' 등 용어 통일
* Wz 상세 정보 리스트에 MergedVersion 사용

### CharaSim
* 장비 툴팁
  * "캐릭터 기준 10레벨 당 체력" 옵션 지원 추가 (Thanks to #80)
  * 안드로이드 외형 (인간형 안드로이드의 경우는 예시 있는 부위만 확인 가능) 지원 추가

### Comparer
* 파일 정보에 관련된 모든 파일 표시 기능 추가

### Patcher
* KMST1125 유형 패치 파일을 사용할 때 패치 전 체크섬 표시 기능 추가

### Avatar
* 진행도 창 모양 변경사항 '동작 모두 내보내기'에 반영

### MapRender
* [KMS 1.2.126, KMS 1.2.352] 몬스터 파크 미니맵에 몬스터 위치 표시 기능 추가
* NPC hideName 속성으로 미니맵에 누락되지 않도록 변경

### 기타
* WzComparerR2 2.2 업데이트 반영

<hr /><em>This discussion was created from the release <a href='https://github.com/KENNYSOFT/WzComparerR2/releases/tag/'>KMS-v4.0.0</a>.</em>

**Comment by nanpuhaha** [comment]
업데이트 감사합니다! 
`DumpXml.lua`, `DumpImages.lua`도 잘 작동하네요!
